client/utils/sanitize.ts

```diff
@@ -0,0 +1,6 @@
+export const checkURL = (str: string) => {
```

Comment:
```suggestion
export const checkURL = (str: string): string => {
```

返り値の型の宣言が抜けてたので足しました。
`check` だと `bool` を返すかも？と思ったりするので、変更を加えそうな `sanitize` の方が良いかもって思いました！